spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -6,6 +6,13 @@
 
   describe '#run_pre_assembly' do
     let(:exp_workflow_svc_url) { Regexp.new("^#{Dor::Config.dor_services.url}/objects/.*/apo_workflows/assemblyWF$") }
+    let(:b) do
+      bc = bundle_context_from_hash('images_jp2_tif')
+      # need to delete progress log to ensure this test doesn't skip objects already run
```

Comment:
There is already a dedicated factory that does this:
https://github.com/sul-dlss/pre-assembly/blob/master/spec/factories/bundle_contexts.rb#L11-L15

Reply:
ah, good catch, thanks!  i was just wrangling the code i saw in front of me, but will switch to using that, and will re-push.